library/core/src/intrinsics/mod.rs

```diff
@@ -2413,35 +2413,14 @@ pub(crate) macro const_eval_select {
             $(#[$compiletime_attr:meta])* $compiletime:block
```

Comment:
Please also update the doc comment above to say that *both* functions are always marked as `inline`.